packages/services/api/src/modules/app-deployments/providers/app-deployments-manager.ts

```diff
@@ -143,11 +188,34 @@ export class AppDeploymentsManager {
       },
     });
 
-    return await this.appDeployments.activateAppDeployment({
+    const result = await this.appDeployments.activateAppDeployment({
       organizationId: token.organizationId,
       targetId: token.targetId,
       appDeployment: args.appDeployment,
     });
+
+    const currentUser = await this.session.getViewer();
```

Comment:
create an app deployment is done via a target access token, so there is no viewer and this will reject.

Reply:
TODO @TuvalSimha: remove audit log creation for app deployments